src/test/java/com/wikia/webdriver/common/core/api/GraphApi.java

```diff
@@ -2,6 +2,7 @@
 
 import com.wikia.webdriver.common.core.XMLReader;
 import com.wikia.webdriver.common.core.helpers.FacebookUser;
+import com.wikia.webdriver.common.core.helpers.GoogleUser;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.common.core.helpers.GoogleUser'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)